templates/navebar.html

```diff
@@ -0,0 +1,4 @@
+<nav class="nav">
```

Comment:
misschien nog even een extra class "bg-light" of "bg-dark" toevoegen. Dan is die toch wat meer visueel aanwezig